app/Models/Comment.php

```diff
@@ -58,15 +58,15 @@ class Comment extends Model
 
     protected $guarded = [];
 
-    public function content(){
+    public function content() {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->